### PR TITLE
Project marianna

### PR DESCRIPTION
**Pull request by mariannahaney**
Pushing 2 custom methods to main

### CODE REVIEW

File: pom.xml
Comment (kZhaanbaev) [inline pom.xml:1]: Why did you delete 3 libraries here?

File: src/main/java/io/techleadacademy/stepDefs/apiSteps/Temp.java
Comment (kZhaanbaev) [inline src/main/java/io/techleadacademy/stepDefs/apiSteps/Temp.java:1]: I recommend to clean up after completing a task. In this case I would assume you used this test to check if it's working. Will be better to delete before pushing since it won't be used in actual test scenarios

File: src/main/java/io/techleadacademy/utils/ApiUtils.java
Comment (kZhaanbaev) [inline src/main/java/io/techleadacademy/utils/ApiUtils.java:1]: Can you also add an overloaded method to createNewModule that accepts a Module object instead of a Map to make it more versatile?

File: src/test/resources/db/modules.feature
Comment (kZhaanbaev) [inline src/test/resources/db/modules.feature:1]: Are these intended changes to push? Just want to make sure you didn't delete existing scenarios on your end and accidentally pushing them

**Review by kZhaanbaev — CHANGES_REQUESTED** [review]

File: pom.xml
Comment (mariannahaney) [reply pom.xml:1]: I was trying to resolve the issue with pom.xml. I will add it to git.ignore and push again
